omod/src/main/java/org/openmrs/module/idgen/rest/resource/LogEntryResource.java

```diff
@@ -96,11 +96,6 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return description;
 	}
 
-	@PropertyGetter("source")
-	public String getSource(LogEntry logentry) {
```

Comment:
Shouldn't this be in a separate pull request as a followup commit for IDGEN-67

Reply:
Fixed